### DISCUSSION TITLE
Can you disable awaitNotInAsync errors?

### DISCUSSION THREAD

**Discussion by fungiboletus** [microsoft/pyright#3593] [Q&A] [answered]
Hei,

I would like to disable `awaitNotInAsync` errors from pyright while waiting for the bug https://github.com/microsoft/vscode-jupyter/issues/10507 to be fixed in vscode-jupyter.

However it doesn't seem to be possible, is this normal?

**Comment by erictraut** [comment, marked as answer]
Yes, there are certain errors that are always emitted by pyright. These are things like syntax errors that will always generate a runtime exception. This falls into that category. You'll need to wait for the other bug fix.

**Reply by fungiboletus** [reply]
Thanks for your answer 😊
